.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,14 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Create a bug report to help us improve
 title: ''
-labels: bug / fix, help wanted
+labels: bug
```

Comment:
I remember @Borda mentioned that all bug reports should have "help wanted" label until it gets assigned to someone.

Reply:
In #9519, @Borda mentions:
> The `help wanted` label shall be used for all bugs

but maybe we can remove it now and add it back if really needed in the future?

Reply:
Let's remove and add on-demand

It doesn't seem like the community has been taking unassigned bug reports at the moment because of this label.